README.md

```diff
@@ -1,16 +1,229 @@
+broken, could the programming language know that? Because Ruby doesn't know
+_what_ to return, it returns `nil`. In this case it's a bit like "No comment."
+
+It certainly makes sense for Ruby to take this attitude, but it'd be horrible
+if Ruby were to take this same attitude toward expression:
+
+```ruby
+1 + 1 #=> Nah, not feeling like adding today, so....nil!
+```
+
+Well, that'd be really unhelpful...and scary! But again, this underscores the
+difference between _expressions_ and _statements_: _expressions_ always return
+values. Statements usually do not.
+
+Let's try a slightly more complex example. You've not been introduced to the `if`
+_statement_ formally yet (soon!), but you should be able to follow this:
+
+```ruby
+if result == 2
+  puts "things match"
+end
+```
+
+You see, the `if` doesn't have a return value, it structures whether or not the
+`puts` expressions will send the `String` constant expression to the screen.
+
+The _expressions_ in this _statement_ are the Boolean-returning equality test
+(`result == 2`) and the constant expression (`"things match"`), but `if...end`
+and `puts` are statements: they don't evaluate an expression; instead they
+decide whether something should happen.
+
+***At its heart, programming is structuring which expressions get evaluated and in
+what order that happens.***
+
+## Identify Three Core Categories of Code Statements
+
+There are three core types of statement that we will cover in this module.
+
+* **Sequence**: What code runs in what order? We make a sandwich before eating it;
+  likewise, certain evaluations will run before others. How is that order
+  determined?
+* **Selection**: Given the default order of sequence, can we deviate from it
+  under certain conditions? How do we do so?
+* **Repetition**: Given a need to something until a something is `true`, or
+  until something has happened a certain number of times, can we change the
+  sequence to make something repeat before moving on?
+
+Let's look at a few examples. Try them out in IRB. We'll cover them more
+formally later.
+
+> **NOTE**: These examples assume you're working in a
+fresh, new IRB session _for each example_. If you define `result` and _don't_
+restart IRB and a later example might assume `result` is `nil`, you might get a
+surprising result. Use `exit` to leave IRB and launch a fresh session if you're
+surprised by a result.
+
+## Provide Example of Sequence Statement
+
+The default **sequence** of code is: "top to bottom, every line, left to right
+as ruled by order of operations"
+
+```ruby
+result = 1 + 1
+puts result #=> 2
+```
+
+You probably have an intuitive belief that the default sequence is this because
+you recognize code as "text" and you probably have the general theory that
+English text is read "top to bottom, left to right."
+
+```ruby
+puts result #=> Error
+result = 1 + 1
+```
+
+This error makes sense, because you're trying to do variable lookup before
+setting the variable.
+
```

Comment:
are selection and repetition statements _also_ sequence statements or is sequence specifically for linear flow?

Reply:
No, there are three top-level classes of statement: (linear flow aka sequence), selection, and repetition. Does that bring up something?

Reply:
ah okay. I think the explanation of sequence may need to be clearer. I think the first sentence in this section makes it sound like this section is talking about control flow rather than what sequence statements are. It makes it seem like they are about order but really, as far as I understand, they're just single statements that maintain normal control flow like `puts`, right? 

Maybe a direct sentence to define sequence statement would help here.  I also think it would be clearer to separate the idea of sequence statements and _sequence_ by using flow control or some other term